### PR TITLE
[WIP] WIP!! vm-sysroot support in CI

### PR DESCRIPTION
**Pull request by sleirsgoevy**

### CODE REVIEW

**Comment by sleirsgoevy** [comment]
bors check

**Comment by sleirsgoevy** [comment]
bors check

**Comment by sleirsgoevy** [comment]
bors try

**Comment by sleirsgoevy** [comment]
bors try

File: scripts/uml-tests.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sleep 10 # wait for jjs-frontend to start up
```
Comment (MikailBag) [inline scripts/uml-tests.sh]: Hack. Can it be avoided?

File: scripts/uml-tests.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+cat > test.cpp << EOF
+#include <iostream>
+int main(){int64_t a,b;std::cin>>a>>b;std::cout<<a+b<<"\\n";}
+EOF
+
+jjs-cli submit --contest trial --filename test.cpp --problem A --toolchain g++
+
+{ sleep 1; poweroff; } &
```
Comment (MikailBag) [inline scripts/uml-tests.sh]: Why  `sleep` here?

Comment (sleirsgoevy) [reply scripts/uml-tests.sh]: I don't remember exactly. Probably it's just to finish the init.d script before shutdown

**Comment by sleirsgoevy** [comment]
bors r=MikailBag

**Comment by sleirsgoevy** [comment]
bors r=MikailBag

**Comment by sleirsgoevy** [comment]
bors r=MikailBag

**Pull request merged**